views/package.json

```
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.8.1",
-        "@oat-sa/tao-item-runner-qti": "1.9.2"
+        "@oat-sa/tao-item-runner-qti": "dev-fix/AUT-2716/be-able-to-associate-pairs-with-audio-video-in-association-interaction as 1.9.2"
```

```suggestion
        "@oat-sa/tao-item-runner-qti": "https://github.com/oat-sa/tao-item-runner-qti-fe#fix/AUT-2716/be-able-to-associate-pairs-with-audio-video-in-association-interaction"
```
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#git-urls-as-dependencies